app/connectors/GetEmploymentExpensesConnector.scala

```diff
@@ -42,7 +43,7 @@ class GetEmploymentExpensesConnector @Inject()(val http: HttpClient, val appConf
     integrationFrameworkCall(integrationFrameworkHeaderCarrier(url, apiVersion))
   }
 
-  private def shouldUse2324(taxYear: Int): Boolean = {
+  private def isYearAfter2324(taxYear: Int): Boolean = {
     taxYear == 2024
```

Comment:
New API should work beyond 2024... ` taxYear >= 2024 `

Reply:
Good one